com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs

```diff
@@ -976,6 +976,8 @@ static void DrawEmissionAdvancedContent(SerializedHDLight serialized, Editor own
                 --EditorGUI.indentLevel;
             }
 
+            EditorGUILayout.PropertyField(serialized.includeForRayTracing, s_Styles.includeLightForRayTracing);
```

Comment:
I think we should show this field only when Raytracing is enabled. What do you think?